qml-ui/main.qml

```diff
@@ -680,388 +680,443 @@ Kirigami.AbstractApplicationWindow {
+                        icon.name: "media-record-symbolic"
+                        icon.width: 24
+                        icon.height: 24
+                        icon.color:  globalRecordButton.currentSequence.activePatternObject && globalRecordButton.currentSequence.activePatternObject.recordLive
+                                     ? "#ff5cf436" // A green with the same values as the red audio record colour below
+                                     : zynqtgui.sketchpad.isRecording ? "#fff44336" : Kirigami.Theme.textColor
+                    }
+
+                    QQC2.Button {
+                        id: globalPlaybackButton
+                        width: Kirigami.Units.gridUnit*4
+                        height: parent.height
+                        onClicked: {
+                            if (zynqtgui.sketchpad.isMetronomeRunning) {
+                                zynqtgui.callable_ui_action_simple("SWITCH_STOP");
+                            } else {
+                                zynqtgui.callable_ui_action_simple("SWITCH_PLAY");
+                            }
+                        }
+
+                        icon.name: zynqtgui.sketchpad.isMetronomeRunning ? "media-playback-stop" : "media-playback-start"
+                        icon.width: 24
+                        icon.height: 24
+                        icon.color: Kirigami.Theme.textColor
                     }
-                } else {
-                    applicationWindow().openRecordingPopup();
                 }
             }
 
-            Kirigami.Icon {
-                width: Kirigami.Units.gridUnit
-                height: width
-                anchors.centerIn: parent
-                source: "media-record-symbolic"
-                color: globalRecordButton.currentSequence.activePatternObject && globalRecordButton.currentSequence.activePatternObject.recordLive
-                    ? "#ff5cf436" // A green with the same values as the red audio record colour below
-                    : zynqtgui.sketchpad.isRecording ? "#fff44336" : "white"
-            }
-        }
-        QQC2.Button {
-            Layout.preferredWidth: Kirigami.Units.gridUnit*4
-            Layout.preferredHeight: Kirigami.Units.gridUnit*2
-            onClicked: {
-                if (zynqtgui.sketchpad.isMetronomeRunning) {
-                    zynqtgui.callable_ui_action_simple("SWITCH_STOP");
-                } else {
-                    zynqtgui.callable_ui_action_simple("SWITCH_PLAY");
-                }
+            Item {
+                Layout.fillWidth: true
             }
 
-            Kirigami.Icon {
-                width: Kirigami.Units.gridUnit
-                height: width
-                anchors.centerIn: parent
-                source: zynqtgui.sketchpad.isMetronomeRunning ? "media-playback-stop" : "media-playback-start"
-                color: "white"
+            Zynthian.StatusInfo {
+                highlighted: zynqtgui.sketchpad.isMetronomeRunning
```

Comment:
Please note that this property did not exist previously, so it depends on https://github.com/zynthbox/zynthbox-qml/pull/16 being accepted.

The highlighted prop can be used to indicate playback state, or that the child popup is open. etc